src/main/java/com/homework/springhomework/exceptions/ExceptionHelper.java

```diff
@@ -0,0 +1,31 @@
+package com.homework.springhomework.exceptions;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+@Slf4j
+@ControllerAdvice
+public class ExceptionHelper {
```

Comment:
Should be called `ExceptionHandler`

Reply:
I can't change the class name. I think this is due to the same name- the class ExceptionHandler is already used (import org.springframework.web.bind.annotation.ExceptionHandler) when I used annotation '@ExceptionHandler'. 
![image](https://user-images.githubusercontent.com/61257154/129486091-881b0cc2-436e-494b-a0e6-ce08b5cc24cf.png)
![image](https://user-images.githubusercontent.com/61257154/129486157-d2ef4b5d-b472-498b-bcd6-a63884871936.png)

Reply:
I've changed the class name to StudentExceptionHandler. Thx :)